app/views/components/_warning_summary.html.erb

```diff
@@ -2,30 +2,27 @@
   title ||= false
   description ||= false
   items ||= []
-  id ||= "app-c-warning-summary-#{SecureRandom.hex(4)}"
+  id ||= id
```

Comment:
is this meant to be nil?

Reply:
🤦‍♂️ yep, good spot.

Reply:
Updated

Reply:
Looks still here - did you push?

Reply:
Yep, put the it seems like I got the same error in two files, now is fixed in both.